src/puppetlabs/ezbake/core.clj

```diff
@@ -699,6 +707,38 @@ Additional uberjar dependencies:
 (defmulti action
   (fn [action & args] action))
 
+(defn munge-deps
+  ([deps]
+   (into {} (concat (for [[k v] deps]
+                      (let [[project-name version] k
+                            dependencies (munge-deps v)
+                            metadata (if (empty? dependencies) {"version" version} {"version" version "dependencies" dependencies })
```

Comment:
Oh I noticed there's already plenty of ambiguous calls to `(str ...)` in this file, so I'd be fine leaving this as `str`.

(Maybe I'm not remembering this correctly, and this is not ambiguous. Oh well)